src/main/java/com/exasol/containers/ExasolContainer.java

```diff
@@ -239,6 +244,23 @@ public String getDriverClassName() {
 
     @Override
     public String getJdbcUrl() {
+        if ((this.clusterConfiguration != null) && (getDockerImageReference().getMajor() >= 7)) {
+            final Optional<String> fingerprint = this.certificateProvider.getSha256Fingerprint();
+            if (fingerprint.isEmpty()) {
+                return getJdbcUrlWithoutFingerprint();
```

Comment:
I think you could add this case to the above if statement

Reply:
I thought about this, but `getSha256Fingerprint()` reads and parses the certificate. I want to avoid doing this if not required.

Reply:
so what's the issue with that?